exporter/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+use std::path::PathBuf;
+use std::process::ExitCode;
+
+use guac::collector::emitter::NatsEmitter;
+
+pub mod exporter;
+
+#[derive(clap::ValueEnum, Debug, Clone)]
+pub enum Events {
+    #[clap(name = "kafka")]
+    Kafka,
+    #[clap(name = "sqs")]
+    Sqs,
+}
+
+#[derive(clap::Args, Debug)]
+#[command(about = "Run the exporter", args_conflicts_with_subcommands = true)]
+pub struct Run {
+    #[arg(short = 'i', long = "index")]
+    pub(crate) index: Option<PathBuf>,
+
+    #[arg(long = "kafka-bootstrap-servers", default_value = "localhost:9092")]
+    pub(crate) kafka_bootstrap_servers: String,
+
+    #[arg(long = "guac-url", default_value = "127.0.0.1:4222")]
+    pub(crate) guac_url: String,
+
+    // Event bus used to communicate with other services.
+    #[arg(long = "events", value_enum, default_value = "kafka")]
+    pub(crate) events: Events,
+
+    #[arg(long = "stored-topic", default_value = "sbom-stored")]
+    pub(crate) stored_topic: String,
+
+    #[arg(long = "indexed-topic", default_value = "sbom-indexed")]
+    pub(crate) indexed_topic: String,
+
+    #[arg(long = "failed-topic", default_value = "sbom-failed")]
+    pub(crate) failed_topic: String,
+
+    #[arg(long = "sync-interval-seconds", default_value_t = 10)]
```

Comment:
I think the exported should listen to 'sbom-indexed' potentially? I think only one of sbom-indexed or sbom-failed should be needed, so this is more 'notified' topic